src/Mvc/Extensions.ApiDescription.Client/src/build/Microsoft.Extensions.ApiDescription.Client.targets

```diff
@@ -0,0 +1,149 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<Project>
+  <!-- Internal settings. Not intended for customization. -->
+  <PropertyGroup>
+    <GenerateOpenApiReferenceCodeDependsOn>
+      _GetMetadataForOpenApiReferences;
+      _GenerateOpenApiReferenceCode;
+      _CreateCompileItemsForOpenApiReferences
+    </GenerateOpenApiReferenceCodeDependsOn>
+    <GenerateServiceFileReferenceCodeDependsOn>
+      _GenerateErrorsForOldItems;
+      _CreateOpenApiReferenceItemsForOpenApiProjectReferences;
+      GenerateOpenApiReferenceCode
+    </GenerateServiceFileReferenceCodeDependsOn>
+  </PropertyGroup>
+
+  <!-- OpenApiProjectReference support. -->
+
+  <ItemGroup>
+    <!--
+      Do not change %(ReferenceOutputAssembly) if project contains duplicate @(ProjectReference) and
+      @(OpenApiProjectReference) items.
+    -->
+    <ProjectReference Include="@(OpenApiProjectReference)" Exclude="@(ProjectReference)">
+      <NoWarn>NU1702</NoWarn>
+      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
+    </ProjectReference>
+    <ProjectReference Update="@(OpenApiProjectReference)">
```

Comment:
Any reason not to inline this with the previous include?

Reply:
Because when `@(ProjectReference)` and `@(OpenApiProjectReference)` overlap we need to get the documents. The previous element is about the non-overlapping case and this covers everything.

Reply:
MSBuild and the SDK don't handle duplicate `<ProjectReference>` items well. This can cause some strange behaviors such as https://github.com/Microsoft/msbuild/issues/2688. Consider making this overlap an error.

Reply:
The two blocks are separate specifically to avoid creating duplicate `<ProjectReference>` items. Why make overlap between `@(ProjectReference)` and `@(OpenApiProjectReference)` an error with that in place?